pkg/scale/scale.go

```diff
@@ -0,0 +1,230 @@
+package scale
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/appscode/go/log"
+	"github.com/appscode/go/types"
+	apps_util "github.com/appscode/kutil/apps/v1beta1"
+	core_util "github.com/appscode/kutil/core/v1"
+	ext_util "github.com/appscode/kutil/extensions/v1beta1"
+	kutil "github.com/appscode/kutil/meta"
+	api "github.com/appscode/stash/apis/stash/v1alpha1"
+	"github.com/appscode/stash/pkg/backup"
+	"github.com/appscode/stash/pkg/util"
+	apps "k8s.io/api/apps/v1beta1"
+	core "k8s.io/api/core/v1"
+	extensions "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type Options struct {
+	Workload       api.LocalTypedReference
+	Namespace      string
+	Selector       string
+	DockerRegistry string // image registry for check job
```

Comment:
do we need DockerRegistry & ImageTag here?

Reply:
I think so. We need to support private registry.

Reply:
Are we creating any new pods from scaledown job? I mean check jobs are created from init-container.

Reply:
It was not necessary. Removed now.